### DISCUSSION TITLE
lettuce execute command problem in Redis cluster environment

### DISCUSSION THREAD

**Discussion by noseew** [redis/lettuce#2183] [Q&A]
Question: Want to know how lettuce executes commands in a Redis cluster environment? Is there a detailed technical documentation for this? I didn't find a detailed explanation in the wiki. Our team plans to use lettuce as a Redis client, so I want to know more about its features

**Comment by mp911de** [comment]
https://github.com/lettuce-io/lettuce-core/wiki/Redis-Cluster#command-routing is a good source to start. Other than that, there's no real magic involved, just sending commands to the right cluster node.
